Tests/test_geo.py

```diff
@@ -3214,7 +3214,8 @@ def test_record_str(self):
 
 Sample_description: Fibroblast cell line derived from a 1 mo
 nth old female with multiple congenital malformations, dysmorphic features, intr
-auterine growth retardation, heart murmur, cleft palate, equinovarus deformity,
+auterine growth retardation, heart murmur, cleft palate, equinovarus deformity, """
+                                          """
```

Comment:
These triple quotes look like an accidental edit?

Reply:
Amended, the test was passing but flake8 failing, so had to insert /t or /n, that one was left behind.